framework/src/main/java/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -470,12 +470,22 @@ public final synchronized void init(ProcessingEnvironment env) {
         // This is used to trigger AggregateChecker's setProcessingEnvironment.
         setProcessingEnvironment(env);
 
+        // Keep in synce with check in checker-framework/build.gradle.
```

Comment:
`synce` -> `sync`.

A third location that we should probably mention in these spots is
https://checkerframework.org/manual/#installation
There it currently says `Requirement: You must have JDK 8 or JDK 11 installed. (The Checker Framework should work with any JDK 8+, but we only test with JDK 8 and JDK 11.)`
which this PR could also bring in line.